examples/resources/genesyscloud_guide_version/resource.tf

```diff
@@ -0,0 +1,23 @@
+resource "genesyscloud_guide_version" "sample-guide" {
+  guide_id    = data.genesyscloud_guide.id
```

Comment:
Whether you are using the reference to `examples/data-sources/genesyscloud_guide/data-source.tf` or `examples/resources/genesyscloud_guide/resource.tf` in your `locals.tf`, your resource path needs to match (`data.genesyscloud_guide.sample_guide.id` or `genesyscloud_guide.sample_guide.id` respectively).

Reply:
It might be good for you to check out `examples/README.md` if you need more info